test/database.test.ts

```diff
@@ -8,8 +8,10 @@ import { Statement } from "../src/statement.js";
 
 describe("database", function () {
   this.timeout("10s");
+
+  const accounts = getAccounts();
   // Note that we're using the second account here
-  const [, wallet] = getAccounts();
+  const wallet = accounts[1];
```

Comment:
`getAccounts()` creates new instances of all of the wallets on each call.  Not a big deal, but this saves a little on memory.